apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/main/java/com/expediagroup/apiary/extensions/events/metastore/listener/KafkaMetaStoreEventListener.java

```diff
@@ -18,50 +18,165 @@
 import static com.expediagroup.apiary.extensions.events.metastore.KafkaProducerProperty.SERDE_CLASS;
 import static com.expediagroup.apiary.extensions.events.metastore.common.PropertyUtils.stringProperty;
 import static com.expediagroup.apiary.extensions.events.metastore.common.io.MetaStoreEventSerDe.serDeForClassName;
-
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
+import static com.expediagroup.apiary.extensions.events.metastore.listener.ListenerUtils.error;
 
 import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hive.metastore.MetaStoreEventListener;
+import org.apache.hadoop.hive.metastore.events.AddIndexEvent;
+import org.apache.hadoop.hive.metastore.events.AddPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.AlterIndexEvent;
+import org.apache.hadoop.hive.metastore.events.AlterPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.AlterTableEvent;
+import org.apache.hadoop.hive.metastore.events.ConfigChangeEvent;
+import org.apache.hadoop.hive.metastore.events.CreateDatabaseEvent;
+import org.apache.hadoop.hive.metastore.events.CreateFunctionEvent;
+import org.apache.hadoop.hive.metastore.events.CreateTableEvent;
+import org.apache.hadoop.hive.metastore.events.DropDatabaseEvent;
+import org.apache.hadoop.hive.metastore.events.DropFunctionEvent;
+import org.apache.hadoop.hive.metastore.events.DropIndexEvent;
+import org.apache.hadoop.hive.metastore.events.DropPartitionEvent;
+import org.apache.hadoop.hive.metastore.events.DropTableEvent;
+import org.apache.hadoop.hive.metastore.events.InsertEvent;
+import org.apache.hadoop.hive.metastore.events.LoadPartitionDoneEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import com.google.common.annotations.VisibleForTesting;
 
-import com.expediagroup.apiary.extensions.events.metastore.common.listener.AbstractMetaStoreEventListener;
+import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableListenerEvent;
 import com.expediagroup.apiary.extensions.events.metastore.common.event.SerializableListenerEventFactory;
 import com.expediagroup.apiary.extensions.events.metastore.common.io.MetaStoreEventSerDe;
-import com.expediagroup.apiary.extensions.events.metastore.common.messaging.MessageTaskFactory;
-import com.expediagroup.apiary.extensions.events.metastore.messaging.KafkaMessageTaskFactory;
+import com.expediagroup.apiary.extensions.events.metastore.messaging.KafkaMessage;
```

Comment:
I think we should have "kafka" somewhere consistent in the package name for all these classes. Maybe `com.expediagroup.apiary.extensions.events.metastore.kafka`? Similar for other classes in this PR to try be consistent and have them grouped together in some way.

Reply:
sounds good, thanks